### ISSUE TITLE
Release 0.9.2-3 Swagger "Try" feature not working

### ISSUE THREAD

**Issue by DerTomm** [smoketurner/dropwizard-swagger#40]
Hi,
I just upgraded dropwizard-swagger in my project from 0.9.2-1 to 0.9.2-3 and I realized two issues afterwards.
1. The defined resource tags by @Api(value="...") are not parsed identically as before. In my case before I had one group "User API" - afterwards Swagger shows the endpoints unter "default".
2. The bigger problem: The "Try" feature is not working anymore. Instead of the result Swagger-UI just shows me the waiting animation forever. Obviously no real request is being sent to the REST server.

Any ideas what could cause this different behaviour between 0.9.2-1 and -3?

**Comment by jplock** [comment]
I'm not sure...I'll play around with it and see if I can replicate it.

**Comment by jplock** [comment]
Could https://github.com/smoketurner/dropwizard-swagger/pull/29 have broken something?

**Comment by DerTomm** [comment]
Am I right that 0.9.2-3 is intended for use with DW 0.9.2? I'll try to create a minimal scenario for replication today.
By the way: Thanks for your efforts developing this very useful Dropwizard component.

**Comment by DerTomm** [comment]
Hi Justin,

attached [DropwizardSwaggerDemo.zip](https://github.com/smoketurner/dropwizard-swagger/files/293225/DropwizardSwaggerDemo.zip) you find a minimal Eclipse project for reproduction of the issue. If you play a little bit with the version setting of dropwizard-swagger in POM.xml (I recommend changing between 0.9.2-1 and 0.9.2-3) you will see the different structure Swagger UI produces.

Regarding the not working "Try" feature in 0.9.2-3 I just found out that this seems to be caused by the defined https server connection. As soon as I change from https on port 9443 to http on 8080 the requests are sent and responses will be shown in Swagger UI.

**Comment by sbgoodm** [comment]
We're experiencing the same issue, and it is due to the Swagger UI defaulting to http in versions prior to 2.2.0. See this issue: https://github.com/swagger-api/swagger-ui/issues/1670

This problem should be resolved in this project with release 1.0.0-2, since the Swagger UI has been upgraded to 2.2.2. However, 1.0.0-2 hasn't been published in Maven Central yet, so I haven't yet been able to verify.

**Comment by sbgoodm** [comment]
After trying out 1.0.0-2 with swagger-ui 2.2.2, the http request issue wasn't resolved, unfortunately. A closer examination of the swagger issues lead me to updating this config in my dropwizard yaml:
```
swagger:
  resourcePackage: our.company.package
  schemes:
    - https
```

Setting the https scheme forces the Swagger UI to fire ajax requests over https. Setting the scheme to 'http' wuld force the UI to fire requests over http. The bug with swagger seems related to having multiple schemes defined - only the first scheme in the list is ever used.
